core/src/avm2/activation.rs

```diff
@@ -554,14 +554,14 @@ impl<'a, 'gc> Activation<'a, 'gc> {
     /// outer scope. If the outer scope is empty, we use the bottom
     /// of the current scope stack instead.
     ///
-    /// A return value of `None` implies that both the outer scope, and
-    /// the current scope stack were both empty.
-    pub fn global_scope(&self) -> Option<Value<'gc>> {
+    /// The verifier guarantees that there is always a global scope
+    /// when this function is called.
+    pub fn global_scope(&self) -> Value<'gc> {
```

Comment:
I kinda thought that avmplus supported this edge case, but I can also see the verifier check... weird. (not an actionable note, just thinking loudly)